docs/source/index.rst

```diff
@@ -1,48 +1,18 @@
-Malcolm
-=======
+MalcolmJS
+=========
+
+|buildstatus| |coverage| |readthedocs| |code analysis: airbnb| |code style: prettier|
```

Comment:
Personally I'd prefer not to have the badges in the docs, as they don't always reflect the version being documented